app/app.js

```diff
@@ -0,0 +1,96 @@
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+
+  //  Create function to convert json file to an html string
+  function toHTML() {
+    const fs = require('fs');
+    const joinPath = require('path.join');
+    const arrayPath = joinPath(__dirname, '/array.json');
+
+    //  Read the file
+    fs.readFile(arrayPath, (err, data) => {
+      //  Parse the data and create variables to be used
+      const parsedData = JSON.parse(data);
+      //  If an error occurs log it
+      if (err) {
+        return res.write('There was an error reading the JSON file');
+      }
+
+      const items = [];
+      let html = '';
+
+      //  Loop through the data object and add the values for "type" to the items array
+      parsedData.forEach((item) => {
+        items.push(item.type.toLowerCase());
+      });
+
+      //  Create loop to convert to html
+      items.forEach((item) => {
+        let element = '';
+
+        //  Create switch statement for values
+        switch (item) {
```

Comment:
Requirements: The logical implementation you have built strips out any possible configuration within your array.json file and replaces it with empty form tags.